packages/expo-modules-core/android/proguard-rules.pro

```diff
@@ -9,6 +9,7 @@
 -keep enum * implements expo.modules.kotlin.types.Enumerable {
   *;
 }
+-keepnames class * implements java.io.Serializable
```

Comment:
having a sense that Serializable is too broad and may keep user's code if they are using Serializable.
- is it possible to keep only expo-module's method parameters?
- or at least maybe we could keep Pair first

Reply:
We could just target `Pair` but if we add more types, it'll be the same issue. It's also mentioned [here](https://www.guardsquare.com/manual/configuration/examples) under `Processing serializable classes
`

Reply:
Do the rules here not just affect `expo-modules-core` or are they merged with the others in the app?

Reply:
i was thinking that would affect the app, please correct me if i'm wrong.

Reply:
I'm not sure to be honest but I'm also not sure if it's a big issue if it does affect the app module. We just don't want to obfuscate the class name. Keeping just `Pair` will work but we'll need to remember to test the rules again if we add more converters.

Reply:
let's investigate whether "keep only expo-module's method parameters" is feasible.